webapp/loggedin/modules/data/validationReport/validationReportView.js

```diff
@@ -0,0 +1,61 @@
+import './validationReportView.scss'
+
+import React from 'react'
+import { connect } from 'react-redux'
+
+import * as RecordValidationReportItem from '@core/record/recordValidationReportItem'
+
+import TableView from '@webapp/loggedin/tableViews/tableView'
+import { appModuleUri, dataModules } from '@webapp/loggedin/appModules'
+
+import { useOnUpdate } from '@webapp/commonComponents/hooks'
+
+import * as SurveyState from '@webapp/survey/surveyState'
+
+import { reloadListItems } from '@webapp/loggedin/tableViews/actions'
+
+import ValidationReportRowHeader from './validationReportRowHeader'
+import ValidationReportRow from './validationReportRow'
+
+const validationReportModule = 'validationReport'
+
+const ValidationReportView = props => {
+  const { surveyCycleKey, reloadListItems, history } = props
+
+  useOnUpdate(() => {
+    reloadListItems(validationReportModule, { cycle: surveyCycleKey })
+  }, [surveyCycleKey])
+
+  const onRowClick = row => {
+    const focusNodeUuid = RecordValidationReportItem.getNodeContextUuid(row)
+    const focusNodeDefUuid = RecordValidationReportItem.getNodeDefContextUuid(row)
+    const recordUuid = RecordValidationReportItem.getRecordUuid(row)
+    const recordEditUrl = `${appModuleUri(
+      dataModules.record,
+    )}${recordUuid}?focusNodeUuid=${focusNodeUuid}&focusNodeDefUuid=${focusNodeDefUuid}`
```

Comment:
i would call the params the ```page``` prefix . e.g. ```pageNodeDefUuid```

Reply:
ok